javascript-view-swapping/main.js

```diff
@@ -0,0 +1,17 @@
+const $tabContainer = document.querySelector('.tab-container');
+const $tabElements = document.querySelectorAll('.tab');
+const $viewElements = document.querySelectorAll('.view');
+
+$tabContainer.addEventListener('click', function (event) {
+  if (event.target.matches('.tab')) {
+    const data = event.target.getAttribute('data-view');
+    for (let i = 0; i < $tabElements.length; i++) {
+      if ($tabElements[i] === event.target) {
+        $tabElements[i].setAttribute('class', 'tab active');
+      } else $tabElements[i].setAttribute('class', 'tab');
```

Comment:
You should put brackets after your `else` statements. It makes it hard to read otherwise.

```
if ($tabElement[i] === event.target) {
  $tabElements[i].setAttribute('class', 'tab active');
} else {
  $tabElements[i].setAttribute('class', 'tab');
}